src/com/itschool/session6/homework/challenge05/Challenge05.java

```diff
@@ -0,0 +1,23 @@
+package com.itschool.session6.homework.challenge05;
+
+public class Challenge05 {
+    public static void main(String[] args) {
+        forLoop();
+        printEvenNumbers();
+    }
+
+    static void forLoop() {
+        for (int i=1; i<=10; i++ ) {
+            System.out.println(i + " ");
```

Comment:
unnecessary concatenation with `" "` since you are displaying `i` in the console on different lines; the space won't be visible anyway. If you want to keep it with this concatenation and display everything on a single line you can replace `System.out.println` with `System.out.print`.

Reply:
solved